script.js

```diff
@@ -1,4 +1,183 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+// Version 2 Comfortable - (Mon - Thu: 2h - 3h)
+/*
+- Player input number of dice, number of player and then select game mode
+- Press start button to display the player and the player's optimal combination
+- The highest / Lowest combination wins
+*/
+const diceInput = document.querySelector("#dice-number");
+const playerInput = document.querySelector("#player-number");
+const gameModeInput = document.querySelector("#game-mode");
+const submitBtn = document.querySelector("#submit-button");
+const outputDiv = document.querySelector("#output-div");
+const resetBtn = document.querySelector("#reset-button");
+resetBtn.style.display = "none";
+
+let gameRecordArr = [];
+let playerCounter = 1;
+let diceRollMsg, winner, winnerMsg;
+let displayMsg = "";
+let numsArr = "";
+let gameBoard;
+
+// Toggle Buttons
+function toggleBtn() {
+  if (resetBtn.style.display == "none") {
+    submitBtn.style.display = "none";
+    resetBtn.style.display = "block";
+  } else {
+    submitBtn.style.display = "block";
+    resetBtn.style.display = "none";
+  }
+}
+
+// Submit button click event
+submitBtn.addEventListener("click", function () {
+  let result = main();
+  submitBtn.disable = true;
+  outputDiv.innerHTML = result;
+});
+
+// Reset button click event
+resetBtn.addEventListener("click", function () {
+  location.reload();
+});
+
+// Roll dice
+function rollDice() {
+  let diceNum = Math.floor(Math.random() * 6) + 1;
+  return diceNum;
+}
```

Comment:
kinda nitpicky but since you're not doing any intermediate calculations after assigning a value to diceNum, then you can actually just return that whole Math.floor(...) portion

alternatively look into arrow functions which were introduced in ES6 (2015 release of javascript, this will allow you to define this function in one single line

```
const rollDice = () => Math.floor(Math.random() * 6) + 1;
```